### PR TITLE
Redirect to Pending Releases Page fixes #1355

### LINKED ISSUE #1355: ship it should redirect to "pending releases" after submitting one
Right now, it simply refreshes the new release page - which is almost never what you want. Typically the next action taken is to start a phase - which happens on the Pending Releases page.

### PR DESCRIPTION
**Pull request by salmanzego**
Fixed Issue of Redirection to the pending releases page when a new release is submitted
fixes #1355

### CODE REVIEW

File: frontend/src/views/NewRelease/index.jsx [hunk 1]
```diff
@@ -394,7 +396,11 @@ export default function NewRelease() {
                 partialVersions,
                 version,
                 buildNumber
-              )
+              ).then(response => {
+                if (response === 201) {
```
Comment (gabrielBusta) [inline frontend/src/views/NewRelease/index.jsx:400]:
```suggestion
                if (!response.error) {
```

Comment (gabrielBusta) [reply frontend/src/views/NewRelease/index.jsx:400]: The action returns the response data, not the response itself.

**Review by gabrielBusta — CHANGES_REQUESTED** [review]

File: frontend/src/views/NewRelease/index.jsx (hunk 1 above)
Comment (salmanzego) [reply frontend/src/views/NewRelease/index.jsx:400]: Oh sorry, I hadn't thought about it. Changed it. Thank you.

**Comment by salmanzego** [comment]
Hai, Can you please review this @gabrielBusta

**Pull request closed without merging**